impl/src/main/java/slack/ThreadMessageHandler.java

```diff
@@ -1,11 +1,6 @@
 package slack;
 
-import api.Answer;
-import api.AnswerResource;
-import api.Question;
-import api.QuestionResource;
-import api.User;
-import api.UserResource;
+import api.*;
```

Comment:
Do we have a sonar rule for this?

Reply:
no, that was probably my editor config at my mac that did something.